### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by roshankern**
This PR has two important documentation updates:

1) Make very clear which code is using multi class models and what code is using single class models in `evaluate models`.
2) Clarify that `folded` phenotypic class is not used throughout this repo.

### CODE REVIEW

**Comment by roshankern** [comment]
This PR is ready for review!

File: 0.download_data/README.md [hunk 1]
```diff
@@ -14,6 +14,34 @@ The version of mitocheck_data used is specified by the hash corresponding to a c
...
+| Elongated           | 110   |
+| ADCCM               | 95    |
+| Anaphase            | 84    |
+| Large               | 79    |
+| Grape               | 74    |
+| Metaphase           | 74    |
+| Folded              | 54    |
+
+**Note**: The `get_features_data()` function (defined in [split_utils.py](../utils/split_utils.py)) used to load the labeled cell dataset excludes cells from the `Folded` phenotypic class when loading the labeled cells.
+In our testing, the low representation of `Folded` cells leads to significantly low classification accuracy for this class.
```
Comment (jenna-tomkinson) [inline 0.download_data/README.md]: Is the low classification accuracy seen in both of the models (multi + single class)?

**Review by jenna-tomkinson — APPROVED** [review]
LGTM! I added one small question to see if it provides more clarity to the README but other than please feel free to merge!

File: 0.download_data/README.md (hunk 1 above)
Comment (roshankern) [reply 0.download_data/README.md]: We only tested for multi class, I'll add a note about this to the readme.

**Pull request merged**